library/global.ml

```diff
@@ -209,10 +183,3 @@ let register_inline c = globalize0 (Safe_typing.register_inline c)
 
 let set_strategy k l =
   GlobalSafeEnv.set_safe_env (Safe_typing.set_strategy (safe_env ()) k l)
-
-let set_reduction_sharing b =
```

Comment:
Why remove this? Isn't it just a wrapper, precisely?

Reply:
I removed it by consistency:
- it was only for the `share_reduction` flag and not for the other `typing_flag`
- there was an asymmetry between the `write` method in the `declare_option` and the `read` method

In conclusion, I evaluated that the benefit of keeping it was not worth the loss of regularity it was introducing.